File: AdminIPBG/AdminIPBG.Web/Modules/Rows/Rows/RowsExcelImportForm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Serenity.ComponentModel;

namespace AdminIPBG.Modules.Rows.Rows
{
    [FormScript("Rows.RowsExcelImport")]

    public class RowsExcelImportForm
    {
        [FileUploadEditor, Required]
        public String FileName { get; set; }
    }
}

File: AdminIPBG/AdminIPBG.Web/Imports/ServerTypings/RowsExcelImportForm.ts
namespace AdminIPBG.Rows {
    export class RowsExcelImportForm extends Serenity.PrefixedContext {
        static formKey = 'Rows.RowsExcelImport';

    }

    export interface RowsExcelImportForm {
        FileName: Serenity.ImageUploadEditor;
    }

    [['FileName', () => Serenity.ImageUploadEditor]].forEach(x => Object.defineProperty(RowsExcelImportForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

